docs/schemas/Cell-Annotation-and-Taxonomy/Aligned_taxonomy_schema.csv

```diff
@@ -0,0 +1,82 @@
+[additional uncontrolled metadata],30bc73fe-764e-4c62-bcbe-4a28eb96cd0a,,,"Additional uncontrolled cell metadata. These are not required, but any additional columns are allowed by all h5ad formats.",,,,,,,,,annotations,
+[cellannotation_set]--parent_cell_set_accession,b2f33d06-5307-45e7-82a8-e758872ee043,,,"ID corresponding to the parent cell_set. If not needed for annotations, definitely needed for tooling.",,,,,,,,,tooling,
+gene,39a71ba6-895b-4e81-8f00-cb7505fc3ae6,,,"A vector of gene symbols. Broadly useful in the community for defining genes but occasionally problematic; called gene_symbol in BKP. More generally, some alignment is needed about whether these or the ensembl_id are used for the gene identifier column (CELLxGENE uses a very specific version of ensembl_id for the INDEX).",,,,,,,,,"data, analysis",
+ensembl_id,0925b6b6-c117-4f6b-964f-f46368127512,,,A vector of corresponding Ensembl IDs for each gene symbol. This is required for disambiguation of gene symbols; called gene_identifier in BKP. This is optional for AIT (but maybe it sholdn't be?).,,,,,,,,,data,
+biotype,3d0fc718-4ca7-4e98-8185-d45212a80415,,,"biotype from the gtf file (e.g., protein_coding); used in BKP and BICAN for filtering of genes (but optional elsewhere)",,,,,,,,,data,
+name,612f27c3-12c5-4db4-b0af-894972c42907,,,Longer gene name from the gtf file; used in BKP (optional for now),,,,,,,,,data,
+[additional gene info],0332caff-8232-45b4-b4d1-079f973a8504,,,"Optional uncontrolled gene info; could include gene length, Genecode IDs, NCBI identifiers, etc.",,,,,,,,,data,
+marker_genes_[…],deddeb0d-0df2-4e53-bf69-21be2c305cd0,,,"A set of logical vectors (T/F) indicating which genes are markers used to build dendrogram, or for other purposes. The [...] part of the name links to additional metadata in the uns. This needs to be UPDATED in AIT to allow multiple marker gene sets; markers currently stored differently in CAP.",,,,,,,,,"annotations, analysis",
+highly_variable_genes,9d00952f-284b-4575-9a94-2e0572cb4abc,,,A logical vector (T/F) indicating which genes are highly variable. Used for correlation-based mapping in scrattch.mapping.,,,,,,,,,analysis,
+dataset_metadata,92e98047-7e37-42bb-9ba7-42dfae3698f5,,,"TBD information about the data set itself. A standard on this is not established (as far as I know?) but could include some combination of these pieces of information recorded for Annotations below: description, dataset_url, title, dataset_doi, author_list, author_name, author_contact, orcid, etc.",,,,,,,,,data,

```

Comment:
some "description" text in this table looks like it is more commentary from the authors, this line is just one of the example

Reply:
Yes, let's change the description here so it doesn't sound like a commentary.

Reply:
fixed.